src/test/java/com/artipie/debian/DebianITCase.java

```diff
@@ -60,106 +46,62 @@
 public final class DebianITCase {
 
     /**
-     * Repository name.
+     * Test deployments.
+     * @checkstyle VisibilityModifierCheck (10 lines)
      */
-    private static final String NAME = "my-debian";
-
-    /**
-     * Temporary directory for all tests.
-     * @checkstyle VisibilityModifierCheck (3 lines)
-     */
-    @TempDir
-    Path tmp;
-
-    /**
-     * Tested Artipie server.
-     */
-    private ArtipieServer server;
-
-    /**
-     * Container.
-     */
-    private TestContainer cntn;
-
-    /**
-     * Storage.
-     */
-    private Storage storage;
-
-    /**
-     * Artipie server port.
-     */
-    private int port;
+    @RegisterExtension
+    final TestDeployment containers = new TestDeployment(
+        () -> TestDeployment.ArtipieContainer.defaultDefinition()
+            .withRepoConfig("debian/debian.yml", "my-debian"),
+        () -> new TestDeployment.ClientContainer("debian")
+            .withWorkingDirectory("/w")
+            .withClasspathResourceMapping(
+                "debian/sources.list", "/w/sources.list", BindMode.READ_ONLY
+            )
+            .withClasspathResourceMapping(
+                "debian/aglfn_1.7-3_amd64.deb", "/w/aglfn_1.7-3_amd64.deb", BindMode.READ_ONLY
+            )
+    );
 
     @BeforeEach
-    void init() throws Exception {
-        this.storage = new FileStorage(this.tmp);
-        this.server = new ArtipieServer(
-            this.tmp, DebianITCase.NAME,
-            new RepoConfigYaml("deb").withFileStorage(this.tmp.resolve("repos"))
-                .withComponentsAndArchs("main", "amd64")
+    void setUp() throws IOException {
+        this.containers.assertExec(
+            "apt-get update successful",
+            new MavenITCase.ContainerResultMatcher(),
+            "apt-get", "update"
+        );
+        this.containers.assertExec(
+            "Failed to install curl",
+            new MavenITCase.ContainerResultMatcher(),
+            "apt-get", "install", "-y", "curl"
         );
-        this.port = this.server.start();
-        final Path setting = this.tmp.resolve("sources.list");
-        Files.write(
-            setting,
-            String.format(
-                "deb [trusted=yes] http://host.testcontainers.internal:%d/my-debian %s main",
-                this.port, DebianITCase.NAME
-            ).getBytes()
+        this.containers.assertExec(
+            "Move debian sources",
```

Comment:
@olenagerasimova it's failure message that prints if assertion fails, similar to `AssertMatcher.assertThat(msg, ...)`